queries/elixir.scm

```diff
@@ -0,0 +1,41 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(arguments) @arguments_list
+(call
+  target: (_) @functionCallee) @functionCall @functionCallee.domain
```

Comment:
Please install [Andreas's VSCode extension](https://marketplace.visualstudio.com/items?itemName=AndreasArvidsson.andreas-talon) to get correct formatting here. We will eventually format in CI but we're not there yet for this file type

Also, would be good to get tests for this scope

Reply:
Thanks! Much nicer with the extension